aws/dms/tasks.yml

```diff
@@ -58,6 +58,11 @@ cron:
 - dashboard.course_scripts
 - dashboard.survey_results
 - dashboard.sections
+- dashboard.section_instructors
+    # remove_columns:
```

Comment:
It appears that this table does not store any sensitive information (just links to entities in other tables), so we don’t need any `remove_columns` entries. Can you take that out? I can then get this change deployed to our data export system.